apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -348,7 +348,8 @@ export class PostgresClient extends BasicDatabaseClient<QueryResult> {
         CASE
           WHEN data_type = 'ARRAY' THEN 'YES'
           ELSE 'NO'
-        END as is_array
+        END as is_array,
+        pg_catalog.col_description(format('%s.%s', table_schema, table_name)::regclass::oid, ordinal_position) as column_comment
```

Comment:
Thanks!
Couple of questions on this function:
- Does it work for Redshift and CockroachDB? They are subclasses of the postgreSQL driver. If the tests fail, you will have to add a `listTableColumns` for their drivers specifically.
- Can you add a test for this in `tests/integration/lib/db/client/postgres.spec.ts`? It should confirm that comments can be added and returned.

Reply:
> Does it work for Redshift and CockroachDB? They are subclasses of the postgreSQL driver. If the tests fail, you will have to add a listTableColumns for their drivers specifically.
- `postgreSQL` and `cockroachDB` both supports `pg_catalog.col_description` (because cockroachDB uses postgreSQL driver)
- However, old version of cockroachDB(v21) [[does not support](https://www.cockroachlabs.com/docs/v21.1/functions-and-operators)](https://www.cockroachlabs.com/docs/v21.1/functions-and-operators) `format` syntax, so unit test error happens(beekeeper use v21 when doing unit test). But, latest version of cockroachDB(v24) [[supports](https://www.cockroachlabs.com/docs/v24.1/functions-and-operators)](https://www.cockroachlabs.com/docs/v24.1/functions-and-operators)  `format` syntax, as it is kind of basic SQL syntax.
- `cockroachDB` supports separate [`[column_comment](https://www.cockroachlabs.com/docs/v21.1/information-schema#columns)`](https://www.cockroachlabs.com/docs/v21.1/information-schema#columns) column in `information_schema.columns`, which `postgreSQL` doesn’t have.
- _In conclusion, I’ll keep `listTableColumns` in `postgresql.ts`, but I’ll write separate function for `cockroach.ts` by using `column_comment`. This can support lower versions of `cockroachDB` and also can use more clear syntax `column_comment` which `cockroachDB` supports._

Reply:
> Can you add a test for this in tests/integration/lib/db/client/postgres.spec.ts? It should confirm that comments can be added and returned.
Done :)